src/components/Home.js

```diff
@@ -1,8 +1,27 @@
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../apis/axiosClient';
 
 function Home() {
   const { status, user } = useAuth();
 
+  // testing axiosInstance request
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axiosInstance
+      .get('/showMe')
+      .then((response) => {
+        setData(response.data);
+        console.log('this is axios instance' + data);
```

Comment:
I wouldn't expect this to be the _axios instance_, I would expect this to be the HTTP response body returned by the axios instance.